### PR TITLE
Custom check v6

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: samples/custom-checks/monitoring3rdparty/Version_6/Sample/ConfigErrorQueue.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus.Config;
+using NServiceBus.Config.ConfigurationSource;
+
+class ConfigErrorQueue : IProvideConfiguration<MessageForwardingInCaseOfFaultConfig>
```
Comment (SimonCropp) [inline samples/custom-checks/monitoring3rdparty/Version_6/Sample/ConfigErrorQueue.cs]: can u use the code api on busconfig

File: servicecontrol/plugins/custom-checks.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ NOTE: It is essential that you deploy this plugin to your endpoint in production
 
 ## NuGets
 
+ * NServiceBus Version 6.x: [ServiceControl.Plugin.Nsb6.CustomChecks](https://www.nuget.org/packages/ServiceControl.Plugin.Nsb6.CustomChecks)
```
Comment (SimonCropp) [inline servicecontrol/plugins/custom-checks.md]: how do we handle this moving forward? ie until it is really release

File: samples/custom-checks/monitoring3rdparty/Version_6/Sample/app.config [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" /></startup>
```
Comment (SimonCropp) [inline samples/custom-checks/monitoring3rdparty/Version_6/Sample/app.config]: startup is redundant

File: samples/custom-checks/monitoring3rdparty/Version_6/Sample/app.config [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="NServiceBus.Core" publicKeyToken="9fc386479f8a226c" culture="neutral" />
```
Comment (SimonCropp) [inline samples/custom-checks/monitoring3rdparty/Version_6/Sample/app.config]: why the redirect?

File: samples/custom-checks/monitoring3rdparty/Version_6/Sample/Monitor.cs [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+    public override Task<CheckResult> PerformCheck()
+    {
+        try
+        {
+            using (HttpClient client = new HttpClient
+            {
+                Timeout = TimeSpan.FromSeconds(3),
+            })
+            using (HttpResponseMessage response = client.GetAsync(url).Result)
```
Comment (SimonCropp) [inline samples/custom-checks/monitoring3rdparty/Version_6/Sample/Monitor.cs]: this cant be awaited?

File: samples/custom-checks/monitoring3rdparty/Version_6/Sample/ConfigAuditQueue.cs [hunk 6]
```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus.Config;
+using NServiceBus.Config.ConfigurationSource;
+
+class ConfigAuditQueue : IProvideConfiguration<AuditConfig>
```
Comment (SimonCropp) [inline samples/custom-checks/monitoring3rdparty/Version_6/Sample/ConfigAuditQueue.cs]: use AuditProcessedMessagesTo

File: samples/custom-checks/monitoring3rdparty/Version_6/3rdPartySystem/3rdPartySystem.csproj [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Program.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="app.manifest" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
```
Comment (SimonCropp) [inline samples/custom-checks/monitoring3rdparty/Version_6/3rdPartySystem/3rdPartySystem.csproj]: remove the redundant comment

Comment (danielmarbach) [inline samples/custom-checks/monitoring3rdparty/Version_6/3rdPartySystem/3rdPartySystem.csproj]: Ok I fix this on the v5 sample then as well

File: servicecontrol/plugins/custom-checks.md (hunk 2 above)
Comment (danielmarbach) [inline servicecontrol/plugins/custom-checks.md]: Is there any harm in mentioning it already?

**Comment by danielmarbach** [comment]
@SimonCropp fixed

File: servicecontrol/plugins/custom-checks.md (hunk 2 above)
Comment (SimonCropp) [inline servicecontrol/plugins/custom-checks.md]: yeah i cant think of anything better

**Pull request merged**